Shared/Realm.Sync.Shared/Permissions/IPermissionObject.cs

```diff
@@ -18,34 +18,34 @@
 
 using System;
 
-namespace Realms.Sync.Permissions
+namespace Realms.Sync
 {
     /// <summary>
     /// Interface that describes the shared base model of all Permission classes.
     /// </summary>
     public interface IPermissionObject
     {
         /// <summary>
-        /// Gets or sets the unique identifier of this object in the Management realm.
+        /// Gets the unique identifier of this object in the Management realm.
         /// </summary>
-        string Id { get; set; }
+        string Id { get; }
 
         /// <summary>
-        /// Gets or sets the creation time of this object.
+        /// Gets the creation time of this object.
         /// </summary>
-        DateTimeOffset CreatedAt { get; set; }
+        DateTimeOffset CreatedAt { get; }
 
         /// <summary>
-        /// Gets or sets when the object was updated the last time.
+        /// Gets when the object was updated the last time.
         /// </summary>
         /// <remarks>
         /// This should be filled by the client with the <see cref="CreatedAt"/>
         /// date and is updated by the server with the current object when the object is processed.
```

Comment:
Should  _updated by the server with the current object when_ be _updated by the server with the timestamp when_